specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2022-08-01/examples/virtualMachineExamples/VirtualMachine_Get_VM_WithInstanceView.json

```diff
@@ -0,0 +1,221 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscription-id}",
+    "resourceGroupName": "myResourceGroup",
+    "api-version": "2022-08-01",
+    "vmName": "myVM",
+    "$expand": "userData,instanceView"
```

Comment:
this line is failing CI check.  $expand is part of the path, so this line should be removed